.github/workflows/unittests.yml

```diff
@@ -23,4 +23,4 @@ jobs:
         export PATH=${PATH}:${GOPATH}/bin
         GO111MODULE=off go get -u github.com/gordonklaus/ineffassign
         GO111MODULE=off go get -u golang.org/x/lint/golint
-        make check-fmt lint test
```

Comment:
why was check-fmt removed?

Reply:
See ab53f3f2b507eff8210e0edcf5bc784038b28563 or https://github.com/cilium/hubble/pull/178#discussion_r388561406

Reply:
Then why wasn't that file deleted? I wouldn't have any questions then :)

Reply:
No file was deleted as part of this PR :thinking:

Reply:
seems like this got more confusing than necessary :) maybe we should have a single target that does all the formatting/linting/vet checks.

Reply:
@michi-covalent See https://github.com/cilium/hubble/pull/181